data/mods/Xedra_Evolved/mutations/paraclesians/salamander_eocs.json

```diff
@@ -1,4 +1,79 @@
 [
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_SALAMANDER_NEAR_FIRE_CHECK",
+    "condition": { "not": { "and": [ { "u_has_effect": "sleep" }, { "math": [ "u_salamander_near_fire", "==", "0" ] } ] } },
+    "effect": [
+      {
+        "run_eocs": [
+          {
+            "id": "EOC_SALAMANDER_NEAR_FIRE_CHECK_2",
+            "condition": {
+              "or": [
+                { "u_is_in_field": "fd_hot_air1" },
```

Comment:
Wouldn't checks for hot air would result in buff being given if you have a vehicle heater turned on?

Reply:
Yes. I'm okay with that since it also checks ambient temperature (via the weather), but if it's enough of a problem I can always edit the heaters in XE to produce legally-distinct-but-equivalent fields.

Reply:
No, not at all, if it's intentional

Reply:
Ideally I'd get rid of all the field nonsense and just be like "There has to be a fire within 5 tiles or the ambient temperature nearby has to be 80F or higher" but temperature detection either detects weather temperature (independent of fields) or body part temperature (which won't go up on Salamanders because they're immune to heat), so there's no way to do it.